test/kubernetes/e2e/features/helm/suite.go

```diff
@@ -81,7 +82,7 @@ func (s *testingSuite) TestApplyCRDs() {
 		if err != nil {
 			return err
 		}
-		if info.IsDir() {
+		if info.IsDir() || info.Name() == "README.md" {
```

Comment:
ah i didn't realize the readme was added in the `crds` dir.  i'd suggest either moving it elsewhere (if there's a good place to put it), or adding the README.md to the [helmignore](https://github.com/solo-io/gloo/blob/main/install/helm/gloo/.helmignore) file so it doesn't get packaged with our helm chart

Reply:
This feels like a good start, but leaves us open to future files causing the same issue. What if instead, we examine the file extension, and if it's yaml we assume it's a CRD, and if it's not we skip it?

Reply:
Yeah I almost did that but I'm kinda okay with this being more strict to remind us to be thoughtful adding things to that folder? Maybe it'll prompt someone to suggest adding it to the `.helmignore` for instance :)